airflow/utils/helpers.py

```diff
@@ -334,7 +334,7 @@ def get_byte_range_file(fl, num_bytes):
         if current_position == 0:
             return None
         fl.seek(-num_bytes, os.SEEK_CUR)
-        data = str(fl.read(num_bytes), 'utf-8', 'ignore')
+        data = fl.read(num_bytes).decode('utf-8')
```

Comment:
There is no need for Python2.7 support. It has been removed from the master branch